client/signup/steps/site-information/index.jsx

```diff
@@ -92,7 +92,7 @@ export class SiteInformation extends Component {
 				return {
 					fieldLabel: translate( 'Address' ),
 					fieldDescription: translate( 'Where can people find your business?' ),
-					fieldPlaceholder: 'E.g., 60 29th Street #343, San Francisco, CA 94110',
+					fieldPlaceholder: 'E.g., 60 29th St, San Francisco, CA 94110',
```

Comment:
See above, re: `e.g`

Reply:
Oooh, good eye. The editors in residence say it should be "E.g.,"